doc/reference/std/misc/list.md

```diff
@@ -359,9 +359,56 @@ Alias for `delete-duplicates` and `delete-duplicates!` ([SRFI 1](https://srfi.sc
 ```
 :::
 
+## delete-duplicates/hash ##
+```scheme
+(delete-duplicates/hash lst
+  [table: (make-hash-table)]
+  [key: identity]
+  [from-end?: #f]) -> list
+
+  lst       := list from which to delete elements
+  table     := a hash-table with suitable test function and initial elements to detect duplicates
+  key       := optional unary procedure to get the to compare item out of a list element
+  from-end? := optional boolean to delete from the end rather than from the start
+```
```

Comment:
This interface description is lovely, but as far as I can tell there is no standardized format to define something like this, the existing docs are quite inconsistent. Out of scope for this PR, but it would be nice to settle on a format and then write it down in a contributors guide.

Reply:
Agreed. However, I'm waiting for drewc's next get documentation before I'm going to worry about that.